2.4.0 Purchase Orders for B2B (#558)
magento/merchdocs: src/customers/account-dashboard-approval-rules.md

---
title: Approval Rules
---

Most companies have order approval rules for purchase orders. Companies can control who gets to create purchase orders and how much they can spend.
For instance:

*  Any PO less than X value is automatically approved.
*  POs that is over X value but less than Q must be approved by Y.
*  Any PO over X value must be approved by Y and Z.
*  Anyone making a PO at the Director level or above are automatically approved.

> Magneto lets you set up these rules to enforce company policies.

Reviewer: Magento

To create rules, users with role permissions have an 'Approval Rules' link in the left nav bar.
Permissions to create rules are set in the Roles and Permissions page in the left nav bar.

> If no rules have been created, the list is empty and the 'Create Rule' button is displayed.

Reviewer 2: ```suggestion
If no rules have been created, the list is empty. Click <span class="btn">Create Rule<span> to create a new rule. 
```

> Click the button to create a new rule.

Reviewer 2: Just remove this line.

> On the Approval Rules form, rules can be Enabled or Disabled.

Reviewer 2: ```suggestion
On the Approval Rules form, any of the created rules can be enabled or disabled.
```

> Note: When assigning a role as an approver, ensure that there is at least one user in that role.

Reviewer 2: This needs to use the note formatting:
`{:.bs-callout-info}`

## Rule types and common options

> Choose the rule type from the dropdown menu.

Reviewer 2: "dropdown menu" is not OK in user docs. I don't know if there is a label here or what this looks like. If the dropdown has a label, use that name to refer to the UI element. 

```suggestion
Choose the **Rule Type**.
```

All rules have common options:

> *  For 'Rule name', provide a short but descriptive name for the Rule: "Orders less than $100". Names must be unique.

Reviewer 2: ```suggestion
*  For **Rule name**, enter a short but descriptive name for the Rule, such as `Orders less than $100`. Names must be unique.
```

> *  Description: A longer explanation of the rule.

Reviewer 2: ```suggestion
*  For **Description**, enter a longer explanation of the rule.
```

> *  Applies to: Choose the company Role(s) that this rule will apply to.

Reviewer 2: ```suggestion
*  For **Applies to**, choose the company role(s) that this rule will apply to.
```

### Requires approval from

Companies can require approvals from one or more users in other roles.

> To require an approval, choose the required approver(s) from the multi-select list.

Reviewer 2: "multi-select list" is not OK for user docs. Use the same type of language/style as the drop down. Does this have a label? I am not sure what this should be because I don't know what it looks like.

Author: Done

### Order Total

> 

Reviewer 2: Should these be steps? Seems a bit random.

Reviewer 2: I committed updates to restructure topic

Approve the PO from the order total, including tax.

> For Order Total amount, choose from:

Reviewer 2: ```suggestion
Choose an **Order Total amount** option:
```

*  is more than
*  is less than
*  is more than or equal to
*  is less than or equal to

Select the currency type and enter the amount.

Select the role(s) that need to approve this type of PO.

> All defined roles will be listed, a "Purchaser's Manager" option.

Reviewer 2: ??? I can't make sense of this one.

Author: Me neither. Removed.

> Click the "Save" button to create the rule.

Reviewer 2: ```suggestion
Click  <span class="btn">Save</span>.
```

### Number of SKUs

> 

Reviewer 2: Is this an example?  Should these be steps?

Reviewer 2: I committed updates to restructure topic

This rule is based on the number of SKUs or unique products in the order.
This is the number of distinct item types, not the number of items being ordered.

> For instance:

Reviewer 2: ```suggestion
For example:
```

*  2 large white shirts
*  3 medium white shirts

> That is 5 items but 2 distinct SKUs.

Reviewer 2: ```suggestion
That is 5 items, but 2 distinct SKUs.
```

> Set the Number of SKUs value:

Reviewer 2: ```suggestion
Set the **Number of SKUs** value:
```

*  is more than
*  is less than
*  is more than or equal to
*  is less than or equal to

> and the quantity of SKUs to test.

Reviewer 2: This needs to be a complete sentence. 

```suggestion
Set the quantity of SKUs to test.
```

### Shipping cost

> Some companies will limit or control shipping costs. Use this rule to approve or disapprove by shipping cost.

Reviewer 2: This seems like another example. 

```suggestion
Some companies need to limit or control shipping costs. The following example rule settings can be used to approve or disapprove a PO based on shipping cost.
```

> Set the Shipping cost value:

Reviewer 2: ```suggestion
Set the **Shipping cost value**:
```

*  is more than
*  is less than
*  is more than or equal to
*  is less than or equal to

> and the shipping amount to test.

Reviewer 2: Complete sentence.

```suggestion
Set the quantity of SKUs to test.
```

Author: Updated.
